src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -786,6 +786,15 @@ def validate_and_apply_inputs_mapping(self, inputs, inputs_mapping) -> List[Dict
         :return: A list of dictionaries containing the resolved inputs for each line in the flow.
         :rtype: List[Dict[str, Any]]
         """
+        if not inputs_mapping:
+            logger.warning(
+                msg=(
+                    "Starting a batch run without column mapping may lead to unexpected results. "
```

Comment:
"batch run" -> "run" since this can also be a evaluation run?

Reply:
Good suggestion, in my opinion, evaluation run is included in batch run.
Will confirm with @thy09